index.js

```diff
@@ -11,40 +11,37 @@ function instrument(options) {
   var instrumenter = new Instrumenter.Instrumenter();
   var captured = false;
 
-  function transform(file) {
-    // If if doesnt match the pattern dont instrument it
+  return through.obj(function(row, enc, next) {
+    if (row.file.indexOf("node_modules") !== -1) {
```

Comment:
Ok, I understand. How did that work before? Are you explicitly including stuff from `node_modules/something`?